src/test/crux-gateway/test-entity.ts

```diff
@@ -43,65 +43,175 @@ describe('CRUX Gateway Entity Tests', function() {
             await user2GatewayForUser1.sendMessage(testmsg);
         });
 
-        it('Test Certificate Creation', async function() {
-                    
-            const testCeritficate = {
-                claim: this.user2.cruxID.toString(),
-                messageId : "123e4567-e89b-12d3-a456-426614174000"
-            };
-            const certificate = await CertificateManager.make(this.user2IDClaim,"123e4567-e89b-12d3-a456-426614174000");
-            expect(certificate.claim).to.equals(testCeritficate.claim);
-            expect(certificate.messageId).to.equals(testCeritficate.messageId);
-        });
+        describe('Certificate Creation', function() {
```

Comment:
Move all certificate tests to separate test file

Reply:
will do it later otherwise, it will be hard to manage files. Because my editor is not detecting test files inside a folder

Reply:
found a solution. Updating